rosidl_typesupport_opendds_cpp/resource/srv__type_support.cpp.em

```diff
@@ -104,24 +113,126 @@ void * create_requester__@(service.namespaced_type.name)(
     DDS::Subscriber_var dds_subscriber,
     allocator_t allocator)
 {
-@# TODO: Implement, considering original code in ffe10f9 or earlier
+  @# TODO: remove next line when ready
+  return nullptr;
+
+  using RequesterType = rosidl_typesupport_opendds_cpp::Requester<
+    @(__dds_request_wrapper_msg_type),
+    @(__dds_response_wrapper_msg_type)
+  >;
+
+  @# Register TypeSupports
+  @(__dds_msg_typesupport_type)RequestWrapperTypeSupport_var tsRequest =
+    new @(__dds_msg_typesupport_type)RequestWrapperTypeSupportImpl;
+
+  if (tsRequest->register_type(dds_participant, "@(__dds_request_wrapper_msg_type)_") != DDS::RETCODE_OK) {
+    RMW_SET_ERROR_MSG("C++ exception during registering of request message type");
```

Comment:
The error message doesn't seem to be describing what actually happened

Reply:
Fixed